### ISSUE TITLE
[Question] Is there any api available for ComfyUI

### ISSUE THREAD

**Issue by aligjahed** [comfyanonymous/ComfyUI#427]
HI, I was wondering if there is any API endpoints available for ComfyUI like the ones in Automatic1111.

**Comment by comfyanonymous** [comment]
Yes. The api is how the frontend communicates with the backend. You can find a short example here: https://github.com/comfyanonymous/ComfyUI/blob/master/script_examples/basic_api_example.py
